modules/op-changing-pipelines-installation-target-namespace.adoc

```diff
@@ -0,0 +1,32 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_content-type: PROCEDURE
+[id="op-changing-pipelines-installation-target-namespace_{context}"]
+= Changing the {pipelines-shortname} installation target namespace
+
+By default, the {pipelines-shortname} is installed in the `openshift-pipelines` namespace.
+You can change the {pipelines-shortname} installation target namespace by editing the `targetNamespace` field in the specificaton of the `TektonConfig` custom resource (CR) to a custom namespace, for example `<my-custom-namespace>`. The {pipelines-shortname} installation is automatically moved to the new namespace specified.
+
+.Example
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+spec:
+  targetNamespace: <my-custom-namespace> <1>
+----  
+
+<1> Replace the variable <my-custom-namespace> your defined target namespace.
+
+[NOTE]
+====
+Do not set the target namespace to `openshift-operators`.
```

Comment:
may be: `openshift-operators` namespace is not allowed as targetNamespace

Reply:
Hello @jkandasa , I have addressed the changes. PTAL